shuffle/cli/src/account.rs

```diff
@@ -13,17 +13,42 @@ use diem_sdk::{
 use diem_types::{
     account_config, chain_id::ChainId, transaction::authenticator::AuthenticationKey,
 };
+use dir::home_dir;
 use generate_key::load_key;
 use shuffle_transaction_builder::framework::encode_create_parent_vasp_account_script_function;
-use std::path::PathBuf;
+use std::path::{Path};
 
-pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
-    let config_path = project_dir.join("nodeconfig/0").join("node.yaml");
+
+pub fn handle(rotate : bool) -> Result<()> {
+    let node_config_path = &(home_dir().unwrap().join(".shuffle"));
+    if !Path::new(node_config_path).is_dir() {
+        panic!("A node hasn't been created yet! Run shuffle -- account first");
+    }
+    let dev_key_filepath = node_config_path.join("dev.key");
+    let new_account_key;
```

Comment:
Here instead of declaring `new_account_key` and assigning to it later, you can directly do both in one step:
```rust
let new_account_key = 
    if rotate {
        generate_key::generate_and_save_key(dev_key_filepath)
    } else {
        ......
    };
```

Reply:
Wow!!! This is neat! Implementing this!